test/ng/directive/formSpec.js

```diff
@@ -420,15 +423,18 @@ describe('form', function() {
       inject(function($timeout) {
         doc = jqLite('<div>' +
                         '<form ng-submit="submitMe()">' +
-                          '<button ng-click="destroy()"></button>' +
+                          '<button type="submit" ng-click="destroy()"></button>' +
```

Comment:
Is this necessary? I think if there's no other button, then the first button is the submit button